pom.xml

```diff
@@ -30,6 +30,12 @@
       <version>2.8.6</version>
     </dependency>
 
+    <dependency>
```

Comment:
Rather use gson (see other comment) than add a new dependency with the same functionality.

Reply:
I don't really understand why, can you please explain?

Reply:
There are a few reasons. The most important ones:

- Each dependency you use loads code into memory, so increases resource usage. You should only add dependencies if they are needed. If you alread have an existing dependency that supplies sufficient functionality it add unnecessary overhead to add another dependency with the same functionality
- Each dependency has the potential to contain bugs and vulnerabilities. The more dependencies you have the more things you need to monitor and update to ensure the code remains valid.
- Using a consistent approach in code makes it easier to maintain. Using a standard approach (also known as a pattern) to deal with a specific class of problems means that once you understand one piece of code, you easily understand what other code does that solves similar problems. If you use the same API for the same type of task, it is easier to understand the code and make future changes.
- Multiple libraries for working with the same format can cause unexpected problems. If you use one library to generate JSON and a different one to read it and they are slightly incompatible, there might be occasional problems that are very difficult to troubleshoot. An example is that in a small number of cases very specific data might be encoded in a way by one library that causes a parsing error in the other library.